pytorch_toolkit/ote/ote/utils/misc.py

```diff
@@ -158,9 +158,9 @@ def run_through_shell(cmd, verbose=True, check=True):
     assert isinstance(cmd, str)
     log_shell_cmd(cmd)
     return subprocess.run(cmd,
+                          verbose,
```

Comment:
please add a condition checking minor version of python interpreter and use `capture_output` if minor is greater than 6. Otherwise ignore `verbose` param since `subprocess.run` does not have `verbose` as an argument.

See: 
https://docs.python.org/3.6/library/subprocess.html#module-subprocess
https://docs.python.org/3.7/library/subprocess.html#module-subprocess